app/_hub/kong-inc/exit-transformer/index.md

```diff
@@ -205,9 +205,9 @@ return function(status, body, headers)
   end
 
   if not headers then
-    headers = { X-Message = "This adds X-Message to an empty set of headers" }
+    headers = { [X-Message] = "This adds X-Message to an empty set of headers" }
```

Comment:
I think this is supposed to have `"`, ie:
```suggestion
    headers = { ["X-Message"] = "This adds X-Message to an empty set of headers" }
```

There are some example in the exit transformer plugin's repo: https://github.com/Kong/kong-plugin-enterprise-exit-transformer/tree/master/examples

Not exactly the same, but the formatting might help.

Reply:
Thanks @lena-larionova I did get a review from Lluis and had not ready for docs review in PR title. I added details to the ticket.

Reply:
I'm not going to show all the ways you can format it at this time.